js/app.js

```diff
@@ -35,10 +35,10 @@ define([
 		var appModel = function () {
 			$.support.cors = true;
 			var self = this;
-			let appRoutes = {};
+			let routes = {};
```

Comment:
as to me I would make it immutable:
`const routes = self.pages.reduce((r, page) => ({...r, ...page.buildRoutes(self)}), {})`